From: "Matteo F. Vescovi" <mfv.debian@gmail.com>
Date: Sat, 28 Apr 2012 10:50:34 +0200
Subject: install_in_usr_share

 * Arch-specific files go to /usr/share
 * Make blender look for systemdir into /usr/share
---
 source/creator/CMakeLists.txt |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/source/creator/CMakeLists.txt b/source/creator/CMakeLists.txt
index 65682fb..8371524 100644
--- a/source/creator/CMakeLists.txt
+++ b/source/creator/CMakeLists.txt
@@ -281,8 +281,8 @@ if(WITH_PYTHON)
 	endif()
 
 	install(
-		DIRECTORY ${CMAKE_SOURCE_DIR}/release/scripts
-		DESTINATION ${TARGETDIR_VER}
+		DIRECTORY ${CMAKE_SOURCE_DIR}/release/scripts/
+		DESTINATION ${CMAKE_INSTALL_PREFIX}/share/blender/scripts/
 		PATTERN ".svn" EXCLUDE
 		PATTERN "__pycache__" EXCLUDE
 		PATTERN "${ADDON_EXCLUDE_CONDITIONAL}" EXCLUDE
